Makefile

```diff
@@ -42,3 +46,10 @@ elf-clean: elf-clean.o
 
 clean:
 	rm -f el elg elf *.o
+
+pub-css:
```

Comment:
I guess this can be simplified by changing target name to `pub.css`. I'll try this as a follow up of this PR